<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QWidget" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QSpinBox" name="m_y_sb">
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Police :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="m_rotation_sb">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Rotation :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="m_x_sb">
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="m_size_sb"/>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLineEdit" name="m_line_edit">
     <property name="placeholderText">
      <string>Entrer votre texte ici</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="KColorButton" name="m_color_pb"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Couleur :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="m_font_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
